test/run/build-rpms.bats

```diff
@@ -1,19 +1,30 @@
 load ../common
 
-@test 'build/install/uninstall RPMs' {
-    skip 'issue #594'
+setup () {
     scope standard
-    prerequisites_ok centos7
     [[ $CHTEST_GITWD ]] || skip "not in Git working directory"
-    if ( ! command -v sphinx-build > /dev/null 2>&1 ); then
-        pedantic_fail 'Sphinx is not installed'
+    if     ! command -v sphinx-build > /dev/null 2>&1 \
+        && ! command -v sphinx-build-3.6 > /dev/null 2>&1; then
+        skip 'Sphinx is not installed'
     fi
+}
+
+@test 'build/install epel7 RPMs' {
+    prerequisites_ok centos7
     img=${ch_imgdir}/centos7
+    image_ok "$img"
+    if [[ -d ${BATS_TMPDIR}/rpmbuild/SOURCES/charliecloud ]]; then
+        rm -rf "${BATS_TMPDIR}/rpmbuild/SOURCES/charliecloud"
+    fi
 
     # Build and install RPMs into CentOS 7 image.
     (cd .. && packaging/fedora/build --install --image="$img" \
                                      --rpmbuild="$BATS_TMPDIR/rpmbuild" HEAD)
+}
 
+@test 'check epel7 RPM files' {
+    prerequisites_ok centos7
+    img=${ch_imgdir}/centos7
     # Do installed RPMs look sane?
     run ch-run "$img" -- rpm -qa "charliecloud*"
```

Comment:
Check for doc package too?

Reply:
Is that not covered by the glob here?

Reply:
Sorry, I was unclear. I was thinking of below, which checks the output of this `rpm -qa`. The doc package is missing from those checks.